Carbon/calculations/calcBottles.js

```diff
@@ -0,0 +1,15 @@
+/*
+    Takes the amount of bottled used in a day that are not recycled
+    Paramater
+    bottles - the bottles wasted today
+    Return
+    CO2 produced in lbs
+
+*/
+//we not using this right now
+const calcBottles = (bottles) =>
+{
+    return Math.round(bottles * .343);
+    //source https://tappwater.co/en/carbon-footprint-bottled-water-2/#:~:text=Based%20on%20the%20most%20cited,592%20km)%20with%20a%20car.   
```

Comment:
SICK, I love you have the source. This will be great for us building out the poster and presentation.